site/js/modules/webui/render.js

```diff
@@ -39,6 +39,80 @@ function cards(cards) {
     `;
 }
 
+// Create and display a web page that follows a standard template.
```

Comment:
"Web page" is a bit too large.
We are not actually rendering a full page (that is what is displayed inside a browser tab).
We are rendering a more conceptual content page (not a formal thing).
So I would just keep it at "page".